hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCreateRecordUtils.scala

```diff
@@ -159,6 +159,55 @@ object HoodieCreateRecordUtils {
           }
         }).toJavaRDD()
 
+      case HoodieRecord.HoodieRecordType.AVRO_BINARY =>
```

Comment:
need to property fix this. there is only one or two lines changed compared to "AVRO" type.